Main.java

```diff
@@ -1,223 +1,220 @@
+                String userInput = input(menuPrompt);
+
+                switch (userInput.toUpperCase()) {
+
+                    case "M":
+                        addMember();
+                        break;
+
+                    case "LM":
+                        outputMembers();
+                        break;
+
+                    case "B":
+                        addBook();
+                        break;
+
+                    case "LB":
+                        outputBooks();
+                        break;
+
+                    case "FB":
+                        fixBooks();
+                        break;
+
+                    case "L":
+                        borrowBook();
+                        break;
+
+                    case "R":
+                        returnBook();
+                        break;
+
+                    case "LL":
+                        outputCurrentLoans();
+                        break;
+
+                    case "P":
+                        runFinesProcess();
+                        break;
+
+                    case "T":
+                        incrementDate();
+                        break;
+
+                    case "Q":
+                        finishedProcess = true;
+                        break;
+
+                    default:
+                        output("\nInvalid option\n");
+                        break;
+                }
+                
+                Library.saveInstanceToFile();
+            }
+        } catch (RuntimeException e) {
+            output(e);
+        }
+
+        output("\nEnded\n");
+    }
+
+    private static void runFinesProcess() {
+        PayFineControl payFineControl = new PayFineControl();
+        new PayFineUI(payFineControl).run();
+    }
+
+    private static void outputCurrentLoans() {
+        output("");
+
+        for (loan loanRecord : libraryService.getCurrentLoansList()) {
+            output(loanRecord + "\n");
+        }
+    }
+
+    private static void outputBooks() {
+        output("");
+
+        for (Book bookRecord : libraryService.getBookList()) {
+            output(bookRecord + "\n");
+        }
+    }
+
+    private static void outputMembers() {
+        output("");
+
+        for (Member memberRecord : libraryService.getMemberList()) {
+            output(memberRecord + "\n");
+        }
+    }
+
+    private static void borrowBook() {
+        BorrowBookControl control = new BorrowBookControl();
+        new BorrowBookUI(control).run();
+    }
+
+    private static void returnBook() {
+        ReturnBookControl control = new ReturnBookControl();
+        new ReturnBookUI(control).run();
+    }
+
+    private static void fixBooks() {
+        FixBookControl control = new FixBookControl();
+        new FixBookUI(control).run();
+    }
+
+    private static void incrementDate() {
+        try {
+            int days = Integer.valueOf(input("Enter number of days: ")).intValue();
```

Comment:
DEFECT 6: separate nested methods in "int days = Integer.valueOf(input("Enter number of days: ")).intValue();"